src/common/components/repository-row/repositoryRow.css

```diff
@@ -1,5 +1,6 @@
 .snapName {
   width: 100%;
+  height: 24px;
```

Comment:
the input height is breaking the layout, a better fix might be to make the rows taller, but I think that's a design change